lib/modules/manager/helmfile/extract.ts

```diff
@@ -118,12 +118,12 @@ export async function extractPackageFile(
 
       if (isOCIRegistry(dep.chart)) {
         res.datasource = DockerDatasource.id;
-        res.packageName = joinUrlParts(repoName, depName);
+        res.packageName = join(repoName, depName);
```

Comment:
I don't like that workaround, it will not work on windows

Reply:
should i use `upath.posix.join`?

Reply:
or is there something im not familiar with? please elaborate what should we do here :D

Reply:
i will return it to + '/' + then its best